paddle/fluid/imperative/tracer.h

```diff
@@ -94,5 +94,9 @@ class Tracer {
   std::unique_ptr<UniqueNameGenerator> generator_;
 };
 
+// To access static variable current_tracer
+std::shared_ptr<Tracer>& GetCurrentTracer();
```

Comment:
Try to return non-const pointer instead of non-const reference.

Reply:
Done, thanks.